app/models/collecting_event.rb

```diff
@@ -994,7 +994,7 @@ def verbatim_map_center(delta_z = 0.0)
     unless verbatim_latitude.blank? or verbatim_longitude.blank?
       lat  = Utilities::Geo.degrees_minutes_seconds_to_decimal_degrees(verbatim_latitude.to_s)
       long = Utilities::Geo.degrees_minutes_seconds_to_decimal_degrees(verbatim_longitude.to_s)
-      elev = Utilities::Geo.distance_in_meters(verbatim_elevation.to_s)
+      elev = Utilities::Geo.distance_in_meters(verbatim_elevation.to_s).to_f
```

Comment:
@jrflood Should we make `distance_in_meters` return .to_f?  Seems logical to do it there.